### PR TITLE
enhance(install): use label selector list to find deployments while installing

### PR DESCRIPTION
**Pull request by daemon1024**
This helps with removing deprecated controller deployments as well which are not installed by karmor anymore

### CODE REVIEW

File: install/install.go [hunk 1]
```diff
@@ -671,8 +672,8 @@ func K8sUninstaller(c *k8s.Client, o Options) error {
 	}
 
 	if err := c.K8sClientset.RbacV1().Roles(o.Namespace).Delete(context.Background(), deployments.KubeArmorControllerLeaderElectionRoleName, metav1.DeleteOptions{}); err != nil {
-		if !strings.Contains(err.Error(), "not foundists") {
-			fmt.Print("Error while uninstalKubeArmor Controller Roleling \n")
+		if !strings.Contains(err.Error(), "not found") {
+			fmt.Print("Error while uninstalKubeArmor Controller Role \n")
```
Comment (Ankurk99) [inline install/install.go:676]:
```suggestion
			fmt.Print("Error while uninstalling KubeArmor Controller Role \n")
```

**Review by Ankurk99 — APPROVED** [review]
LGTM

File: install/install.go [hunk 2]
```diff
@@ -671,8 +672,8 @@ func K8sUninstaller(c *k8s.Client, o Options) error {
 	}
 
 	if err := c.K8sClientset.RbacV1().Roles(o.Namespace).Delete(context.Background(), deployments.KubeArmorControllerLeaderElectionRoleName, metav1.DeleteOptions{}); err != nil {
-		if !strings.Contains(err.Error(), "not foundists") {
-			fmt.Print("Error while uninstalKubeArmor Controller Roleling \n")
+		if !strings.Contains(err.Error(), "not found") {
```
Comment (nyrahul) [inline install/install.go:675]: not foundists! :-))

**Review by nyrahul — APPROVED** [review]

File: install/install.go (hunk 2 above)
Comment (daemon1024) [reply install/install.go:675]: `sed` can sedly cause these sometimes :P

**Pull request merged**